cheri/cheri_insts.sail

```diff
@@ -358,16 +358,11 @@ function clause execute (CIncOffset(cd, cb, rt)) =
   checkCP2usable();
   let cb_val = readCapReg(cb);
   let rt_val = rGPR(rt);
+  let (success, newCap) = incCapOffset(cb_val, rt_val);
   if cb_val.tag & cb_val.sealed & (rt_val != 0x0000000000000000) then
     raise_c2_exception(CapEx_SealViolation, cb)
-  else
-  {
-    let (success, newCap) = incCapOffset(cb_val, rt_val);
-    if success then
-        writeCapReg(cd, newCap)
-    else
-        writeCapReg(cd, int_to_cap(to_bits(64, getCapBase(cb_val)) + rt_val))
-  }
+  else if success then writeCapReg(cd, newCap)
```

Comment:
`writeCapReg` should be on new line for consistency with rest of code base. Same with all other if statements you've edited. I prefer the spaced out version.